examples/r-shiny/README.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Create a Dashboard with Shiny"
+linkTitle: "Create a Dashboard with Shiny"
+weight: 3
+description: Use Shiny to create simple Dashboard and deploy it in Saturn Cloud
+---
+## What is a Dashboard?
+Dashboards is a tool where you can view you raw data in form of meaningful visualisations. It transforms data to charts, graphs and tables that help in analysis of information, defining KPIs  and eventually taking data centric decisions. 
+## What is Shiny?
+<a href="https://shiny.rstudio.com/" target='_blank' rel='noopener'>Shiny</a> is a widely used package for building interactive web applications in R.  Shiny is easy to use and flexible. It integrates nicely with CSS themes, htmlwidgets and JavaScript actions.
+You can create dashboards or run machine learning models through web apps etc without any prior knowledge of web development tools, using Shiny package. 
+Shiny app structure constitutes : 
+
+**User Interface**: Responsible for creating the layout of your web app, like panels, textbox, dropdown etc. This is basically what users sees at their end.\
+**Server**: Constitutes logic which will display content in the webapp as per the inputs given by user from UI.  \
+**ShinyApp**: This function launches the shiny app as per UI/server pair.
+
+## Objective
+In the example below we are building a dashboard as R script (`.R` file) using Shiny. For this example we are creating a histogram for sale prices as per selected number of bins. The data for same is taken from [Kaggle](https://www.kaggle.com/c/house-prices-advanced-regression-techniques/data).
+
+## Building a Dashboard
+First we will create an R script which installs shiny and ggplot2 packages from CRAN. Let's name this file as `setup.R`.
```

Comment:
ggplot2 is included in the `saturncloud-rstudio` image, so I would mention that and remove the explicit installation of ggplot2